pkg/api/grpcServer.go

```diff
@@ -61,9 +61,119 @@ type (
 )
 
 var (
```

Comment:
would it be worth having this in a separate file -- something like //pkg/api/globalwordlist.go ?  It would be cool if we could autogen from the global_dictonary.yaml too.